### DISCUSSION TITLE
tariffInformation.demandWindow doesn't return the right value

### DISCUSSION THREAD

**Discussion by upsuper** [amberelectric/public-api#157] [General]
In the schema, for `tariffInformation.demandWindow` it says
> Is this interval currently in the demand window? Only available if the site in on a demand tariff

I'm currently on tariff EA116 of Ausgrid. Based on Ausgrid's [Tariff Structure Statement](https://cdn.ausgrid.com.au/-/media/Documents/Regulation/Reports-plans/Ausgrid-approved-TSS-2019-24.pdf?rev=bdcad85222e34653a4e1df2177a433cf):
![image](https://github.com/amberelectric/public-api/assets/333750/80ff991e-2eb8-4e0e-a832-90017c6e8357)

However, in the data I fetch from the public API, basically every interval says `demandWindow: true`, for example:
```json
  {
    "type": "ActualInterval",
    "date": "2023-09-11",
    "duration": 30,
    "startTime": "2023-09-10T15:30:01Z",
    "endTime": "2023-09-10T16:00:00Z",
    "nemTime": "2023-09-11T02:00:00+10:00",
    "perKwh": 15.88945,
    "renewables": 8.764,
    "spotPerKwh": 8.39249,
    "channelType": "general",
    "spikeStatus": "none",
    "tariffInformation": {
      "demandWindow": true
    },
    "descriptor": "veryLow"
  },
```
and
```json
  {
    "type": "ForecastInterval",
    "date": "2023-09-13",
    "duration": 30,
    "startTime": "2023-09-12T16:30:01Z",
    "endTime": "2023-09-12T17:00:00Z",
    "nemTime": "2023-09-13T03:00:00+10:00",
    "perKwh": 17.44116,
    "renewables": 4.5828204631863905,
    "spotPerKwh": 9.8472,
    "channelType": "general",
    "spikeStatus": "none",
    "tariffInformation": {
      "demandWindow": true
    },
    "descriptor": "veryLow"
  },
```

The value is clearly wrong I believe.

**Comment by sstteevvee** [comment]
I'm also on EA116, and have noticed the exact same thing with my actual and forecast intervals.

I've been with Amber for only 21 days so far, so was going to wait until my first bill to see how any demand fees appeared there.

**Reply by sstteevvee** [reply]
My first bill finally arrived, which includes a demand charge which is definitely based on a limited subset of the intervals (i.e. it is for a peak usage that is under my highest peak ever) -- so can confirm that the API response is not correct, nor due to my account type.

**Comment by upsuper** [comment]
Probably posting it as an issue is more efficient. Just posted #161.

**Reply by herbi3** [reply]
The data retrieved shows 2pm on the 11th of September, which for that tariff the demand window starts at 2pm until 8pm based on the tariff information, so the data retrieved is true?

Can you show more data say for 10am and see if it's the same?

**Reply by upsuper** [reply]
`2023-09-11T02:00:00+10:00` means 2am I believe? 2pm should be 14:00.
